getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+   "id": "23716db9-d8c8-4682-b662-0de8c185bd07",
+   "metadata": {},
+   "source": [
+    "`%pip install -U langchain langchain_openai`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f634643-841b-47fb-ba55-ff058e863528",
+   "metadata": {},
+   "source": [
+    "#### Install Packages in a Virtual Environment (Optional)\n",
+    "Set up a virtual environment by going to your project directory and executing the following command. This will create a new virtual environment in a folder named `.venv`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "59fa41c8-3535-4d9b-9c30-59e3416ff875",
+   "metadata": {},
+   "source": [
+    "**MacOS/UNIX**\n",
+    "- `python3 -m venv .venv`\n",
+    "\n",
+    "**Windows**\n",
+    "- `py -m venv .venv`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4679ec04-7fe4-49db-9887-2a821943f029",
+   "metadata": {},
+   "source": [
+    "#### Activating the Virtual Environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74c7218d-e68c-4af6-8600-9bb523490392",
+   "metadata": {},
+   "source": [
+    "To use the virtual environment, you must first activate it by executing the following command."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "38cba946-94fd-445c-b23e-eea68661826d",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "**MacOS/UNIX**\n",
+    "- `source .venv/bin/activate`\n",
+    "\n",
+    "  \n",
+    "**Windows**\n",
+    "- `.venv\\Scripts\\activate`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "994e5683-d8d4-4f37-945e-19b1d9aab967",
+   "metadata": {},
+   "source": [
+    "#### Deactivating the Virtual Environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c481e4e-4c39-46ba-98c1-92448da66e93",
+   "metadata": {},
+   "source": [
+    "If you want to leave the virtual environment in MacOS/UNIX and windows, simply execute the following command in the terminal:\n",
+    "\n",
+    "`deactivate`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "46149db2-b280-45d5-b910-df1db63040bf",
+   "metadata": {},
+   "source": [
+    "#### Set all required Environment Variables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4bf3454d",
+   "metadata": {},
+   "source": [
+    "Create a `.env` file and store your credentials like follows:\n",
+    "\n",
+    "`AZURE_OPENAI_API_KEY` = <\"Your key here\">\n",
+    "\n",
+    "`AZURE_OPENAI_API_ENDPOINT` = <\"Your endpoint here\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7723ed6a",
+   "metadata": {},
+   "source": [
+    "\n",
+    "#### Or use dotenv to set and load all required env variables into/from your `.env` file.\n",
+    "\n",
+    "`%pip install python-dotenv`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "471ae21c",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(True, 'COMPLETIONS_MODEL', 'Your model here')"
```

Comment:
This isn't an OpenAI naming convention, this is just how we were using the env variables in our notebook. I think more commonly the Open AI docs use `DEPLOYMENT_MODEL` but because we use both the gpt model and embeddings model I felt it might be confusing to set one as `DEPLOYMENT_MODEL` and the reset it to embeddings later on.